server/src/main/xsd/gateway-config-201616.xsd

```diff
@@ -1501,6 +1501,16 @@
                     <documentation>The maximum redirects http connector follows</documentation>
                 </annotation>
             </element>
+            <element name="http.max.authentication.attempts" maxOccurs="1" minOccurs="0" type="nonNegativeInteger">
+                <annotation>
+                    <documentation>The maximum responses to a 401 a http connector will make</documentation>
+                </annotation>
+            </element>
+            <element name="http.authenticator" maxOccurs="1" minOccurs="0" type="gateway:CollapsedString">
+                <annotation>
+                    <documentation>The authenticator the http connector should authenticate with</documentation>
```

Comment:
This should of been removed in refactoring. Now we use default Authenticator on JVM.  In the future, we might have our own Authenticator class you can implement.  This will allow us to scope the Authenticator to a service.